### ISSUE TITLE
modal does not open if state is set to true  initially

### ISSUE THREAD

**Issue by DeveloperAlly** [bodyflex/react-native-simple-modal#17]
eg

```
constructor() {
   super();
   this.state = {
      open.true
    };
 }
```

**Comment by cenkakin** [comment]
+1, that is also a problem we encountered

**Comment by cenkakin** [comment]
I opened a PR for this: https://github.com/bodyflex/react-native-simple-modal/pull/18

closes #17

**Comment by httnn** [comment]
FYI @AlisonWonderlandApps, this has now been fixed in v4.0.2. thanks to @cenkakin for the PR!
